test/integration/default/inspec/default_spec.rb

```diff
@@ -66,12 +66,14 @@
   end
 
   %w[foo bar].each do |service|
-    unless os.debian?
+    unless os.debian? || os.linux?
```

Comment:
debian is linux

Reply:
No, from what I understand from https://docs.chef.io/inspec/resources/os/#osfamily-helpers linux does not include debian, neither it includes redhat/centos, that is why they are still tested.

Reply:
hum